lib/uplinkc/testdata/object_test.c

```diff
@@ -152,7 +152,12 @@ void handle_project(ProjectRef project) {
         ObjectInfo *object;
         for (int i=0; i < objects_list.length; i++) {
             object = &objects_list.items[i];
-            require(true == array_contains(object->path, object_paths, num_of_objects));
+            const char *objNumStr = strtok(object->path, "test-object");
+            int objNum = atoi(objNumStr);
+            size_t expected_size = pow(10, (double)objNum-1) * 1024 * 5;
```

Comment:
Could you add a comment above this line like you have in the uploader section?: `// NB: 5KB, 50KB, 500KB, 5000KB`